vllm/engine/arg_utils.py

```diff
@@ -1834,8 +1834,11 @@ def add_cli_args(parser: FlexibleArgumentParser,
         parser.add_argument('--disable-log-requests',
                             action='store_true',
                             help='Disable logging requests.')
-        from vllm.platforms import current_platform
-        current_platform.pre_register_and_update(parser)
+        # Skip to avoid triggering platform detection
+        import sys
+        if not any(arg in sys.argv for arg in ["-h", "--help"]):
```

Comment:
we can move this line to `utils.py`, to avoid duplicating it. you might also want to consider `-v` for showing the version.